compressor/seata-compressor-zstd/src/main/java/org/apache/seata/compressor/zstd/ZstdUtil.java

```diff
@@ -24,6 +24,8 @@
  */
 public class ZstdUtil {
 
+    public static final int MAX_COMPRESSED_SIZE = 4 * 1024 * 1024;
```

Comment:
直接定死这个配置值了吗？不需要开放一个配置项吗？
Is this configuration value set directly, or is there an option to configure it?

Reply:
Not open for now; 4MB is quite a large size. The variable data in Seata mainly comes from the primary key values in the branch registration. If we estimate based on a primary key size of 128, it can hold around 32,768 primary key values.

Reply:
ok